fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/BooleanLiteral.java

```diff
@@ -0,0 +1,42 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.nereids.trees.expressions;
+
+import org.apache.doris.nereids.types.BooleanType;
+
+
+/**
+ * Represents Boolean literal
+ */
+public class BooleanLiteral extends Literal {
+
+    public static final BooleanLiteral TRUE_LITERAL = new BooleanLiteral(true);
+    public static final BooleanLiteral FALSE_LITERAL = new BooleanLiteral(false);
```

Comment:
How about simplifying `TRUE_LITERAL` and `FALSE_LITERAL` to `TRUE` and `FALSE`.

Reply:
done